frontend/src/views/camp/navigation/mobile/NavMobileSidebar.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <v-navigation-drawer
+    :value="value"
+    app
+    right
+    width="300"
+    @input="$emit('input', $event)"
+  >
+    <div class="d-flex flex-column fill-height">
+      <div class="d-flex flex-column align-center blue-grey darken-4 white--text py-6">
+        <v-icon x-large>$vuetify.icons.ecamp</v-icon>
+        <h2 class="text-h6 text-center">eCamp</h2>
+      </div>
+      <v-divider class="blue-grey darken-2" />
+      <v-list>
+        <SidebarListItem
+          :title="user.displayName"
+          :subtitle="user.profile().firstname + ' ' + user.profile().surname"
+          :to="{ name: 'profile', query: { isDetail: true } }"
+        >
+          <template #pre>
+            <v-list-item-avatar>
+              <user-avatar :user="$auth.user()" />
+            </v-list-item-avatar>
+          </template>
+        </SidebarListItem>
+        <SidebarListItem
+          :title="$tc('views.camps.title', 2)"
```

Comment:
Also here. Anything in `views.` should match the actual component structure where the translation is used.